### ISSUE TITLE
Accessibility: MAS05A: Keyboard: Loss of focus in Options window on focusing the combo boxes

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/RTVS#3339]
**Platform:**
Windows 10 RS1
Visual Studio Enterprise 2017 Preview
Version: 15.0.26340.0

**Repro Steps:**
1. Launch Visual Studio
2. Hit "Alt+R", to open "Rtools" menu
3. Using arrow keys navigate to "Options" menu item and hit enter
4. Using tab and arrow keys navigate to the combo boxes under "Debugging", "General", "Help", "History", "HTML", "Logging", "Markdown", "R Engine" and "Workspace" expand collapse buttons and observe the visual focus indicator

**Expected Result:**
1. Focus should go the 'Ok' button directly on tabbing from the combo box 
2. Focus should go to the whole combo box i.e.  Edit and button.

**Actual Result:**
On tabbing from 'Evaluate Active bindings' combo box, focus goes to open button and then to 'OK'

**Note**: On tabbing focus moves to the edit field, and on the next tab focus moves to the dropdown button of the combo box and also the focus is not visible on the button. On shift tabbing, focus enters into the left pane tree elements

**User Impact:**
Users with motor/cognitive impairments who rely on keyboard will not know that the focus is on the entire combo box. Users may assume that the focus is only on the text element.

![kb_options](https://cloud.githubusercontent.com/assets/22956130/24295387/8f42db2a-10c0-11e7-9ed0-3b3f711df574.png)

**Comment by MikhailArkhipov** [comment]
This is standard IDE grid. Opened VSTS 399983
